nomination/url_handler.py

```diff
@@ -148,7 +151,11 @@ def add_url(project, form_data):
 def add_metadata(project, form_data):
     summary_list = []
     # Get/Add a nominator
-    nominator = get_nominator(form_data)
+    try:
+        nominator = get_nominator(form_data)
+    except http.Http404:
+        raise http.Http404
```

Comment:
Is there really a point to catching this exception just to raise an identical one immediately? Couldn't we just _not_ use a try/except and have the same result?

Reply:
Yes, that makes sense.

Reply:
This also shows up at 134-137.